mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -217,7 +217,7 @@ def __statistics_overview(self, series, col):
                 text_series = text_series.explode()
                 text_series = text_series[~text_series.str.match('\s*')]
                 data[f'{col}/word_distribution'] = (
-                    text_series.value_counts().head(VALUE_COUNT_LIMIT).todict()
+                    text_series.value_counts().head(VALUE_COUNT_LIMIT).to_dict()
```

Comment:
Let's remove the code of the word_distribution in this PR. Let's add them back in another PR with unit test.

Reply:
I removed the entire block of code that is from a different branch, I'll add it back in which it is ready to ship.